src/Server/OutputParser.cs

```diff
@@ -85,6 +101,28 @@ public static string Parse(char[] buffer,int bufferPos, TerminalOptions terminal
                 else token += buffer[i];
             }
 
+            if (isError)
+            {
+                if (!terminalOptions.UseANSI)
+                    return $"ERROR MESSAGE:{AnsiSequences.NewLine}{result}{AnsiSequences.NewLine}";
+                
+                var firstline = $"{AnsiSequences.ForegroundRed}{new string('X', terminalOptions.Width)}";
```

Comment:
nit: prefer camelCase to uppercase `L` on `Line` for these in both warnings/errors sections?